contribs/gmf/less/mobile-nav.less

```diff
@@ -44,6 +44,30 @@ main {
   }
 }
 
+.search-overlay {
+  /* shadow layer visible when search is active */
+  position: absolute;
+  z-index: @above-menus-index;
+  height: 100%;
+  width: 100%;
```

Comment:
Using less deferent units maps the addition directly with less possible in more cases.
And I prefer using vw and vh against % because we shouldn't worry about the parent div size.
finally your right that in this case it don't change anything :-)